src/Operation/Factory/SubresourceOperationFactory.php

```diff
@@ -113,7 +113,7 @@ private function computeSubresourceOperations(string $resourceClass, array &$tre
                 $operation['identifiers'][$identifier] = [$rootResourceClass, $identifiers[$identifier][1] ?? $identifier, true];
                 $operation['operation_name'] = sprintf(
                     '%s_%s%s',
-                    RouteNameGenerator::inflector($operation['property'], $operation['collection'] ?? false),
+                    RouteNameGenerator::inflector($operation['property'], $operation['collection']),
```

Comment:
same as previous comment, are we sure collection is always set?

Reply:
`$subresource->isCollection()` is defined to return a boolean and not a null value, that why phpstan returns an error. So either we change `isCollection` or this condition.

Reply:
in that case it makes sense.